### DISCUSSION TITLE
Is it possible to open source the code from the nicegui website?

### DISCUSSION THREAD

**Discussion by ponponon** [zauberzeug/nicegui#2056] [Q&A] [answered]
### Question

<img width="1440" alt="图片" src="https://github.com/zauberzeug/nicegui/assets/38725104/210585cf-f9c6-4053-9e3d-90aab0344641">

It seems that the official website of nicegui is also written using nicegui, I hope to get the code of the official website

Because I hope to learn nicegui better through a large and complete project, thank you

**Comment by rodja** [comment, marked as answer]
It's already available. Check `main.py` and `website/`.
